dev-tools/packaging/templates/deb/postinstall.sh.tmpl

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+systemctl daemon-reload 2> /dev/null
```

Comment:
This might be more work, so optional for now, but we should really be using `invoke-rc.d` scripts.  Those scripts handle picking the "right" init system.

https://www.debian.org/doc/debian-policy/ch-opersys.html#interfacing-with-init-systems

Reply:
I created an issue so we don't lose track of it: https://github.com/elastic/beats/issues/36213